dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -570,98 +670,108 @@ public void run(){
     @GET
     @Path ("/cancelIntegrityProcess/{params:.*}")
     @Produces (MediaType.APPLICATION_JSON)
-    public Response cancelIntegrityProcess ( @Context HttpServletRequest httpServletRequest, @Context final HttpServletResponse httpServletResponse, @PathParam ("params") String params ) throws JSONException {
-
-        StringBuilder responseMessage = new StringBuilder();
-
-        InitDataObject initData = webResource.init(params, httpServletRequest, httpServletResponse, true, null);
-        Map<String, String> paramsMap = initData.getParamsMap();
+    public Response cancelIntegrityProcess(@Context final HttpServletRequest httpServletRequest,
+                                           @Context final HttpServletResponse httpServletResponse,
+                                           @PathParam("params") final String params ) {
+        final StringBuilder responseMessage = new StringBuilder();
+        final InitDataObject initData = webResource.init(params, httpServletRequest, httpServletResponse, true, null);
+        final Map<String, String> paramsMap = initData.getParamsMap();
 
         //Validate the parameters
-        String endpointId = paramsMap.get( "endpoint" );
-        if ( !UtilMethods.isSet( endpointId ) ) {
-            Response.ResponseBuilder responseBuilder = Response.status( HttpStatus.SC_BAD_REQUEST );
-            responseBuilder.entity( responseMessage.append( "Error: " ).append( "endpoint" ).append( " is a required Field." ) );
-
+        final String endpointId = paramsMap.get("endpoint");
+        if (!UtilMethods.isSet(endpointId)) {
+            final Response.ResponseBuilder responseBuilder = Response.status(HttpStatus.SC_BAD_REQUEST);
+            responseBuilder.entity(
+                    responseMessage
+                            .append("Error: ")
```

Comment:
Codacy found an issue: [StringBuffer (or StringBuilder).append is called 3 consecutive times with literals. Use a single append with a single combined String.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)